src/models/communities.js

```diff
@@ -0,0 +1,71 @@
+import { DataTypes } from 'sequelize';
+
+import Sequelize from '../sequelize';
+
+const communities = Sequelize.define('communities',
+  {
+    organizers: {
+      type: DataTypes.INTEGER,
+      allowNull: false,
+    },
+    members: {
+      type: DataTypes.INTEGER,
+      allowNull: false,
+    },
+    name: {
+      type: DataTypes.STRING,
+      allowNull: false,
+    },
+    content_types: {
+      type: DataTypes.STRING,
+      allowNull: false,
+    },
+    description: {
+      type: DataTypes.STRING,
+      allowNull: false,
+    },
+    image_id: {
+      type: DataTypes.INTEGER,
+      allowNull: false,
+    },
+    tags: {
+      type: DataTypes.STRING,
+    },
+    website: {
+      type: DataTypes.STRING,
+    },
+    rules: {
+      type: DataTypes.STRING,
+    },
+  },
+  {
+    timestamps: true,
+    paranoid: true,
+    underscored: true,
+  });
+
+const initialize = (models) => {
+  models.communities.belongsTo(models.images, {
+    as: 'image',
+    foreignKey: {
+      name: 'image_id',
+      allowNull: false,
+    },
+  });
+  models.communities.hasMany(
+    models.discussions, {
+      as: 'discussions',
+      foreignKey: 'community_id',
+      sourceKey: 'id',
+    },
+  );
+  models.users.belongsToMany(models.communities, { as: 'userId', through: 'community_user', foreignKey: 'user_id' });
```

Comment:
kafa karışıklığına neden olmamak için `model.user` ile başlayan ilişki fonksiyonlarını `src/models/user.js` dosyasına yazmalısın bu sayede `user` modelini okuyanlar `communities` ile ilişkisini o dosyadan da anlayabilir.